src/net_processing.cpp

```diff
@@ -1786,13 +1817,15 @@ void PeerManagerImpl::UpdatedBlockTip(const CBlockIndex *pindexNew, const CBlock
     }
 
     // Relay to all peers
-    m_connman.ForEachNode([&vHashes](CNode* pnode) {
-        if (!pnode->CanRelay()) {
-            return;
-        }
-        LOCK(pnode->cs_inventory);
+    m_connman.ForEachNode([this, &vHashes](CNode* pnode) {
+        if (!pnode->CanRelay()) return;
```

Comment:
19829: add TODO and/or mark it `partial`?

Reply:
Resolved in latest push